utils/release_helper.py

```diff
@@ -0,0 +1,786 @@
+    if not is_next_release_in_progress(repo):
+        if args.message:
+            message = get_release_end_message(repo)
+            get_communication_channels('finish')
+            print(message)
+    else:
+        print('Please, ensure the all steps in the Release process are concluded.')
+
+
+def parse_arguments():
+    '''Call argparse to process input parameters and return parsed args'''
+    parser = argparse.ArgumentParser(
+        description="Automate Github tasks included in the Release Process.",
+        epilog="Example: release_helper.py -c ~/secrets.ini -r ComplianceAsCode/content stats",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument(
+        '-c', '--creds-file', default='~/secret.ini',
+        help="INI file containing Github token.")
+    parser.add_argument(
+        '-r', '--repository', action='store', default='ComplianceAsCode/content',
+        help="Project repository name.")
+
+    subparsers = parser.add_subparsers(dest='subcmd', required=True)
+    subparsers.add_parser(
+        'stats', help="Return information about the repository current state.")
+
+    cleanup_parser = subparsers.add_parser(
+        'cleanup', help="Cleanup after completing a release.")
+    cleanup_parser.add_argument(
+        '-b', '--branch', action='store_true',
+        help="Remove temporary branch used during stabilization phase.")
+
+    release_prep_parser = subparsers.add_parser(
+        'release_prep', help="Prepare the repository for the next release.")
+    release_prep_parser.add_argument(
+        '-c', '--contributors', action='store_true',
+        help="Update the contributors lists. Do it before creating the stabilization branch.")
+    release_prep_parser.add_argument(
+        '-b', '--branch', action='store_true',
+        help="Create the stabilization branch for the next release.")
+    release_prep_parser.add_argument(
+        '-m', '--milestone', action='store_true',
+        help="Create the next milestone and close the current milestone.")
+    release_prep_parser.add_argument(
+        '-i', '--issues', action='store_true',
+        help="Move Open Issues with a milestone to the next milestone.")
+    release_prep_parser.add_argument(
+        '-p', '--prs', action='store_true',
+        help="Move Open PRs with a milestone to the new milestone.")
+
+    release_parser = subparsers.add_parser(
+        'release', help="Tasks to be done during the stabilization phase.")
+    release_parser.add_argument(
+        '-t', '--tag', action='store_true',
+        help="Show commands to properly tag the branch and start the release.")
+    release_parser.add_argument(
+        '-m', '--message', action='store_true',
+        help="Prepare a message to communicate the stabilization process has started.")
+    release_parser.add_argument(
+        '-b', '--bump-version', action='store_true',
+        help="Bump the project version in *master* branch.")
```

Comment:
Ideally the version bump should be done right after the stabilization is branched.
 
Maybe this could be moved to the release_prep sub command?

Reply:
I don't get the argument for the single source for truth, after we branch out stabilization both branches will diverge. Master will get new PRs and move forward, stabilization will slowdown and only get fixes.

Main reason to bump the version in `master` is to be able easily differentiate one build from the other.

Reply:
Identification of the build is a good argument for this @yuumasato . I am convinced to move it.